contracts/interfaces/IERC20Votes.sol

```diff
@@ -0,0 +1,9 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+interface IERC20Votes {
```

Comment:
As a comment, this interface could be imported from OZ (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/utils/IVotes.sol).

Did you manually create it for some other reason?

Reply:
Ah yeah, not sure why I didn't just use the OZ one. Fixed just now